docker-compose.dev.yml

```diff
@@ -20,6 +20,9 @@ services:
         CLAM_AV:
         CVAT_DEBUG_ENABLED:
     environment:
+      # Use this with CVAT_DEBUG_ENABLED to avoid server response timeouts
+      # Remove the same setting in the docker-compose.yml for this override to work
+      # DJANGO_MODWSGI_EXTRA_ARGS: '--request-timeout 0 --queue-timeout 0 --socket-timeout 1000000'
```

Comment:
After upgrading to uvicorn these options are no longer available, I think that similar behavior can be achieved by changing `*timeout` options in [nginx configuration](https://github.com/opencv/cvat/blob/develop/cvat/nginx.conf), but need to check.

https://nginx.org/en/docs/http/ngx_http_proxy_module.html

Reply:
Ok, ASGI turned out to be working normally in debug without these settings. However there is no debug wrapper in the `asgi.py`, so I added one. Do we still need `cvat/wsgi.py`?

Reply:
Ok, ASGI turned out to be working normally in debug without these settings. However there is no debug wrapper in the `asgi.py`, so I added one. Do we still need `cvat/wsgi.py`?

Reply:
We no longer need `cvat/wsgi.py`, it seems I forgot to delete the file in my PR

Reply:
Done